pkg/crc/network/proxy.go

```diff
@@ -69,12 +69,12 @@ func NewProxyConfig() (*ProxyConfig, error) {
 	return &config, nil
 }
 
-func getProxyFromEnv(proxyScheme string) string {
-	p := os.Getenv(proxyScheme)
-	if p == "" {
-		p = os.Getenv(strings.ToUpper(proxyScheme))
+func getEnv(key string) string {
+	p := os.Getenv(key)
+	if p != "" {
+		return p
 	}
-	return p
+	return os.Getenv(strings.ToUpper(key))
```

Comment:
Looking for both `env_var` and `ENV_VAR` in the environment is actually something I've only seen for proxy variables, not something I'd do in general.

Reply:
I dropped the rename :+1: